script.js

```diff
@@ -0,0 +1,160 @@
+// @ts-check
+
+import { composeCreateOrUpdateTextFile } from "@octokit/plugin-create-or-update-text-file";
+import prettier from "prettier";
+import YAML from 'yaml'
+
+const { parseDocument, parse, stringify, Document } = YAML
+
+const isYamlFile = (fileName) => {
+  const yamlFilePattern = /^.*\.ya?ml$/;
+  return yamlFilePattern.test(fileName);
```

Comment:
I think you can short this to one line, it's clear enough what it does because of the function name. You can shorten the regex and only check for the file extension, too.

```suggestion
  return /\.ya?ml$/.test(fileName);
```